plugins/wacom-usb/fu-wac-device.c

```diff
@@ -784,6 +785,14 @@ fu_wac_device_add_modules(FuWacDevice *self, GError **error)
 			fu_device_set_name(FU_DEVICE(module), name);
 			fu_device_set_version_from_uint16(FU_DEVICE(module), ver);
 			break;
+		case FU_WAC_MODULE_FW_TYPE_TOUCH_ID7:
+			module = fu_wac_module_touch_id7_new(FU_DEVICE(self));
+			name = g_strdup_printf("%s [Touch Module (ID7)]",
```

Comment:
I think the stuff in [] probably should be in description rather than title.

Reply:
I had used the Bluetooth ID6 as an example for the naming part of this submodule, if this is something we want to change should we change the other examples too?

Reply:
Yeah I think so. Generally we want it to be useful and not confusing to people that see it.